assets/prototype/infrastructure/services/contextProviders/ContextProviders.js

```diff
@@ -13,7 +14,9 @@ import { RelationsProvider } from './RelationsProvider';
 const ContextProviders = ({ children, eventId }) => (
 	<ToastProvider>
 		<ApolloProvider>
-			<RelationsProvider eventId={eventId}>{children}</RelationsProvider>
+			<RelationsProvider eventId={eventId}>
+				<EventProvider eventId={eventId}>{children}</EventProvider>
+			</RelationsProvider>
 		</ApolloProvider>
```

Comment:
This is conflicting with `EDTR/master` because #1975 was just merged which added a Create ConfigProvider here as well
(smart of me to have two different people work on separate issues that both affect the same block of code eh? What could go wrong?)

Now on top of that conflict, I think we want to have the `EventProvider` on the outside of this stack and then have the `RelationsProvider` retrieve the `eventId` from the Apollo cache since that is going to be our "single source of truth". As well, after looking at this code and thinking about things a bit, I think we want to also rename this context provider to `EventEditorEventIdProvider` because we will very likely need an `EventProvider` for providing details about any event (for use in other admin pages and/or the site frontend)

But before you do that, you want to pull the latest changes to `EDTR/master` and then rebase this branch onto that one...

**BUT**

We want to avoid the conflict so here's what you could do (although you are welcome to handle things differently as long as the final result is the same):

1. rename your local branch for this PR to something like `FET/create-event-provider-2`
2. pull latest changes for `EDTR/master` 
3. cut a new branch from `EDTR/master` and give it the current name for this branch: `FET/create-event-provider`
4. copy the code over from `FET/create-event-provider-2` into the new branch
5. moving `EventProvider` to the outside of the stack, ie: have it wrap `RelationsProvider`:

```jsx
<EventProvider eventId={eventId}>
    <ConfigProvider>
        <RelationsProvider>{children}</RelationsProvider>
    </ConfigProvider>
</EventProvider>
```

6. then you can change `RelationsProvider` so that it is not getting the Event ID directly from the DOM, but is instead getting it from the EventProvider, but to do that...
7. create a custom hook called `useEventEditorEventId` that imports the EventContext, implements `useContext` to get the event id, and then returns it
8. then import this new hook into `assets/prototype/domain/eventEditor/containers/relations/useRelationsManager.js` and use it to get the event id
9. update `useRelationsManager` and `RelationsProvider` accordingly
10. force push your changes for this branch (set the origin to the same one this PR)

if the above is too much of a pain in the but, then you can also just rebase this onto master and just deal with the conflict since it won't be difficult to resolve

My apologies for the run around with this.

Now one more thing, after the above is all done, you will need to address the issue addressed in this comment: https://github.com/eventespresso/event-espresso-core/issues/1955#issuecomment-562208734